aireview/src/aireview/infrastructure/github_client.py

```diff
@@ -87,12 +95,11 @@ async def get_file_content(
                 data={"ref": ref}
             )
             response.raise_for_status()
-            return base64.b64decode(response.json()["content"]).decode("utf-8")
+            return response.json()
         except Exception as e:
             logger.error(f"Failed to fetch file content for {path}: {str(e)}")
```

Comment:
**style**: Silent failure in get_file_content(). Consider either re-raising the exception or returning a specific error value with proper documentation